sqlite/src/wherecode.c

```diff
@@ -1836,8 +1836,13 @@ Bitmask sqlite3WhereCodeOneLoopStart(
     ** the cursor. In this case it is important to do the full evaluation,
     ** as the result of the expression may not be NULL, even if all table
     ** column values are.  https://www.sqlite.org/src/info/7fa8049685b50b5a
+    **
+    ** Also, do not do this when processing one index an a multi-index
```

Comment:
typo "one index on a multi-index OR clause"

Reply:
I'll have to fix this upstream to avoid merge conflicts.